README.md

```diff
@@ -14,12 +14,48 @@ Our current prototype implementation can be found here:
 
 Limitations include:
 * LevelZero backend support only.
-* Accessors and reductions are currently not supported.
+
+#### Implementation Status
```

Comment:
Would it make sense to clarify again that this is L0 only? And should we add the implementation of the emulation mode for other backends?